compiler-core/src/type_/error.rs

```diff
@@ -789,6 +791,20 @@ pub enum UnknownValueConstructorError {
     },
 }
 
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum UnknownValueConstructorErrorSituation {
+    /// A value using ```let``` is assigned to a variable having a capital case.
+    UnknownConstructorName,
+}
+
+impl UnknownValueConstructorErrorSituation {
+    pub fn description(&self) -> Option<&'static str> {
```

Comment:
What's this for? I couldn't see it used anywhere

Reply:
To make the coding pattern similar to the Unification Errors, I just implemented a rather raw function. Could be removed if it's not necessary.

Reply:
If it's not used then don't include it 👍